### PR TITLE
Document what to do if a class is not generic at runtime

### PR DESCRIPTION
**Pull request by ilevkivskyi**
Fixes https://github.com/python/mypy/issues/5667

### CODE REVIEW

File: docs/source/common_issues.rst [hunk 1]
```diff
@@ -496,6 +496,42 @@ Here's the above example modified to use ``MYPY``:
...
+
+   from queue import Queue
+
+   class Tasks(Queue[str]):  # TypeError: 'type' object is not subscriptable
+       ...
+
+   results: Queue[int] = Queue()  # TypeError: 'type' object is not subscriptable
+
+To avoid these errors while still having precise types one can either use
+string literals or ``typing.TYPE_CHECKING``:
```
Comment (JukkaL) [inline docs/source/common_issues.rst]: "string literals" is a bit vague. Maybe reword as "string literal types".

File: docs/source/common_issues.rst [hunk 2]
```diff
@@ -496,6 +496,42 @@ Here's the above example modified to use ``MYPY``:
...
+.. code-block:: python
+
+   from queue import Queue
+
+   class Tasks(Queue[str]):  # TypeError: 'type' object is not subscriptable
+       ...
+
+   results: Queue[int] = Queue()  # TypeError: 'type' object is not subscriptable
+
+To avoid these errors while still having precise types one can either use
```
Comment (JukkaL) [inline docs/source/common_issues.rst]: Style nit: use "you" instead of "one" for consistency.

File: docs/source/common_issues.rst [hunk 3]
```diff
@@ -496,6 +496,42 @@ Here's the above example modified to use ``MYPY``:
...
+string literals or ``typing.TYPE_CHECKING``:
+
+.. code-block:: python
+
+   from queue import Queue
+   from typing import TYPE_CHECKING
+
+   if TYPE_CHECKING:
+       BaseQueue = Queue[str]
```
Comment (JukkaL) [inline docs/source/common_issues.rst]: Maybe add descriptive comment here -- this is only processed by mypy.

File: docs/source/common_issues.rst [hunk 4]
```diff
@@ -496,6 +496,42 @@ Here's the above example modified to use ``MYPY``:
...
+   else:
+       BaseQueue = Queue
```
Comment (JukkaL) [inline docs/source/common_issues.rst]: Another descriptive comment could be helpful here -- this is not seen by mypy but will be executed at runtime.

**Comment by ilevkivskyi** [comment]
@JelleZijlstra @JukkaL thanks for review! I pushed an update.
